modules/op-using-incoming-webhook-with-pipelines-as-code.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="using-incoming-webhook-with-pipelines-as-code_{context}"]
```

Comment:
@sounix000 incoming webhook we have enhanced  the feature and latest doc is available here https://nightly.pipelines-as-code.pages.dev/docs/guide/incoming_webhook/

Is it possible to change according that

Reply:
I couldn't find much difference. Can you please highlight the specific parts?